protocol/app/prepare/full_node_prepare_proposal_test.go

```diff
@@ -1,220 +1,67 @@
 package prepare_test
 
 import (
+	"bytes"
 	"testing"
+	"time"
 
+	gometrics "github.com/armon/go-metrics"
 	abci "github.com/cometbft/cometbft/abci/types"
+	"github.com/cometbft/cometbft/libs/log"
 	sdktypes "github.com/cosmos/cosmos-sdk/types"
-	"github.com/dydxprotocol/v4-chain/protocol/app/prepare"
-	"github.com/dydxprotocol/v4-chain/protocol/mocks"
-	keepertest "github.com/dydxprotocol/v4-chain/protocol/testutil/keeper"
-	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
-	perpetualtypes "github.com/dydxprotocol/v4-chain/protocol/x/perpetuals/types"
-	pricestypes "github.com/dydxprotocol/v4-chain/protocol/x/prices/types"
-	"github.com/stretchr/testify/mock"
+	"github.com/dydxprotocol/v4-chain/protocol/app/flags"
+	testApp "github.com/dydxprotocol/v4-chain/protocol/testutil/app"
 	"github.com/stretchr/testify/require"
 )
 
 // TestFullNodePrepareProposalHandler test that the full-node PrepareProposal handler always returns
 // an empty result.
 func TestFullNodePrepareProposalHandler(t *testing.T) {
-	tests := map[string]struct {
-		txs      [][]byte
-		maxBytes int64
+	t.Cleanup(gometrics.Shutdown)
 
-		pricesResp    *pricestypes.MsgUpdateMarketPrices
-		pricesEncoder sdktypes.TxEncoder
+	conf := gometrics.DefaultConfig("testService")
+	sink := gometrics.NewInmemSink(time.Hour, time.Hour)
```

Comment:
I don't think adding them as constants will make sense since they aren't something we care whether they are consistent within the application or across tests.

I don't expect these tests to run longer than an hour so retaining the metrics for an hour makes sense to me.